src/language_features/kak_lsp_custom.rs

```diff
@@ -0,0 +1,318 @@
+use crate::{
+    context::*, language_features::hover::editor_hover, markup::escape_brace, types::*, util::*,
```

Comment:
probably better to stick to the one-line-per `use` statement when importing our own modules,
so they are easier to grep. I'm still trying to figure when to use start imports

Reply:
The way the imports are formatted should not be human edited. I think rustfmt has some common options and we should let rustfmt handle import styles.

See https://rust-lang.github.io/rustfmt

e.g. `imports_layout`, `imports_indent` etc.

Reply:
The only _stable_ Rustfmt option in that regard is `reorder_imports = "yes"/"no"`. Granularity/grouping is still left untouched and has to be done manually (or semi-manually with something like rust-analyzer's (un)merge imports actions).